insights/parsers/du_usage.py

```diff
@@ -0,0 +1,60 @@
+"""
+du - Command
+============
+
+Module for the processing of output from the ``du`` command stored in
+file.  The base class ``DBDiskSpace`` provides all of the functionality.
```

Comment:
As we are considering this as a base class, and this may not be database always, should we remove `DB` prefix from `DBDiskSpace`? May be we can name it `DirSpace` or `DirUsage`.

Reply:
I have changed it as DirUsage.

Reply:
I have again modified it as DiskUsage, and changed all names as well which contained db prefix.